torch/csrc/jit/passes/onnx/shape_type_inference.cpp

```diff
@@ -1074,6 +1274,22 @@ void ProcessTimeSeriesNode(Node* n) {
   }
 }
 
+void ProcessUnsqueezeNode(Node* n) {
+  if (TensorTypePtr output_type = n->output(0)->type()->cast<TensorType>()) {
+    if (output_type->dim().has_value()) {
+      size_t rank = static_cast<size_t>(output_type->dim().value());
+      if (rank == 1) {
+        if (ConstantValueMap::HasShapeValue(n->input(0)->debugName())) {
```

Comment:
I think we can combine these 3 if statements into one to reduce nesting:
`if (output_type->dim().has_value() && output_type->dim().value() == 1 && ...) {`